src/main/java/org/hillel/homework/constant/Constant.java

```diff
@@ -0,0 +1,42 @@
+package org.hillel.homework.constant;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum Constant {
+    FAREWELL("Bye!"),
+    NAME_PROMPT("Please provide name: "),
+    COST_PROMPT("Please provide cost: "),
+    TRY_AGAIN("Please provide a valid value"),
+    DESCRIPTION_PROMPT("Please provide description: "),
+    ERROR("Something went wrong"),
```

Comment:
This will work but will cause a lot of issues whe error happen. The error should contain as much useful information as possible: what is the error, what parameters, method the error happen